tests/helpers/shared-test-setups/classic-components.ts

```diff
@@ -11,12 +11,7 @@ const codemodOptions: CodemodOptions = {
 };
 
 const context: Context = {
-  entities: new Map([
-    ['navigation-menu', new Set(['.hbs', '.ts'])],
-    ['products/product/card', new Set(['.hbs', '.js'])],
-    ['products/product/image', new Set(['.hbs', '.ts'])],
-    ['tracks', new Set(['.hbs', '.js'])],
-  ]),
+  entities: new Map(),
```

Comment:
An empty Map is expected, as none of the components in `classic-components` are supported.